webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.test.js

```diff
@@ -0,0 +1,9 @@
+import { testComponentSnapshotsWithFixtures } from '../../../common/testHelpers';
+import PageLayout from './PageLayout';
+import { pageLayoutMock } from './PageLayout.fixtures';
+
+const pageLayoutFixtures = {
+  'render audits page': pageLayoutMock,
```

Comment:
I think there are more cases to cover with this test.
Each props combination will produce a different snapshot.

* With customBreadcrumbs
* With no breadcrumbs
* With toastNotifications on/off
* With searchable on/off
* With toolbarButtons